cluster/examples/kubernetes/ceph/toolbox.yaml

```diff
@@ -17,43 +17,24 @@ spec:
     spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
-      - name: rook-ceph-tools
-        image: rook/ceph:master
-        command: ["/tini"]
-        args: ["-g", "--", "/usr/local/bin/toolbox.sh"]
-        imagePullPolicy: IfNotPresent
-        env:
-          - name: ROOK_ADMIN_SECRET
-            valueFrom:
-              secretKeyRef:
-                name: rook-ceph-mon
-                key: admin-secret
-        securityContext:
-          privileged: true
-        volumeMounts:
-          - mountPath: /dev
-            name: dev
-          - mountPath: /sys/bus
-            name: sysbus
-          - mountPath: /lib/modules
-            name: libmodules
-          - name: mon-endpoint-volume
-            mountPath: /etc/rook
-      # if hostNetwork: false, the "rbd map" command hangs, see https://github.com/rook/rook/issues/2021
-      hostNetwork: true
+        - name: rook-ceph-tools
```

Comment:
I wouldn't expect these lines to change. Was more whitespace added? If so, can you revert it so only the expected lines show as changed?

Reply:
It's strange that this change is not being shown in my vscode